kvmagent/kvmagent/plugins/baremetal_v2_gateway_agent.py

```
@@ -835,16 +836,16 @@ def _create_grub_configuration(self, instance_obj, volume_drivers):
                     nbd_id = volume_driver.nbd_id
                     break
 
-            status,output = commands.getstatusoutput('guestmount --ro -a /dev/nbd%s -m /dev/sda2 %s' % (nbd_id, tempdir))
+            status,output = subprocess.getstatusoutput('guestmount --ro -a /dev/nbd%s -m /dev/sda2 %s' % (nbd_id, tempdir))
```

Ruff (0.15.2)

[warning] Unpacked variable `status` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)